### ISSUE TITLE
Build fails because solution references nonexistent project and missing skins.xml file

### ISSUE THREAD

**Issue by Ban3** [AirVPN/airvpn-client#38]
Project `UI.WpfWeb.Windows` not found in the source tree:

```
airvpn-client/src/Eddie_VS2015.sln (default targets) ->
(Build target) ->

airvpn-client/src/Eddie_VS2015.sln: error : Could not find the project file 'airvpn-client/src/UI.WpfWeb.Windows/UI.WpfWeb.Windows.csproj'
```

Missing `skins.xml`:

```
airvpn-client/src/Eddie_VS2015.sln (default targets) ->
(Build target) ->
airvpn-client/src/Lib.Forms/Lib.Forms.csproj (default targets) ->
/usr/lib/mono/4.5/Microsoft.Common.targets (GenerateResources target) ->

	/usr/lib/mono/4.5/Microsoft.Common.targets: error : Tool exited with code: 1. Output: Error: Invalid ResX input.
Position: Line 246, Column 5.
Inner exception: Could not find a part of the path "airvpn-client/src/Lib.Forms/Resources/skins.xml".
```

Command used to build: ` xbuild /p:Configuration="Release" /p:Platform="x64" src/Eddie_VS2015.sln`

**Comment by Ban3** [comment]
Another missing file: `airvpn-client/src/Lib.Forms/Skin/SkinReference.cs`

**Comment by neeksor** [comment]
I am running into the same issue..

**Comment by Ban3** [comment]
I'm closing this as this was fixed in bfecdbf
